tests/helm_inflator.sh

```diff
@@ -56,7 +56,7 @@ assert_contains_string out.yaml "name: extra-args-redis-master"
 
 helm_testcase "kpt_helm_inflator_imperative_pipeline"
 kpt fn source example-configs |
-  kpt fn run --mount type=bind,src="$(pwd)/${CHARTS_SRC}",dst=/source --image gcr.io/kpt-functions/helm-inflator:"${TAG}" --as-current-user -- local-chart-path=/source/zookeeper name=my-zookeeper | 
+  kpt fn run --mount type=bind,src="$(pwd)/${CHARTS_SRC}",dst=/source,rw=true --image gcr.io/kpt-functions/helm-inflator:"${TAG}" --as-current-user --network -- local-chart-path=/source/zookeeper name=my-zookeeper | 
```

Comment:
The UX here doesn't make sense IMO. At this point, this should fetch everything from the network instead of mounting host volume.

Reply:
This test case is for using local charts in pipeline. There is another case for remote charts in pipeline.

Reply:
Understood. I don't see someone using this flow since it's too cumbersome, instead they would either:

1) Use Helm CLI for local charts
2) Use function for fetching remote packages
3) Use the hermetic function when it's available.